pages/common/minikube.md

```diff
@@ -0,0 +1,25 @@
+# minikube
+
+> Run Kubernetes on your local machine for free.
```

Comment:
Perhaps dropping the `for free` sounds better.

Reply:
I agree. Something like: `Tool to run Kubernetes locally`, sounds better?